### ISSUE TITLE
Test other loss functions

### ISSUE THREAD

**Issue by knkski** [knkski/atai#10]
Right now we're using [categorical cross-entropy](https://github.com/knkski/atai/blob/fabc3ac0400e176eb265fb34c9061bdf44e4f2ae/train.py#L94), which appears to work fine. However, we should investigate [the alternatives](https://keras.io/losses/) and graph the results.
